queries.sql

```diff
@@ -8,3 +8,35 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = 't';
 SELECT * FROM animals WHERE name NOT LIKE '%Gabumon';
```

Comment:
- [ ] As it is a [requirement ](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md)for this project, " `Add a column species of type string to your animals table. Modify your schema.sql file.` ", kindly add a species column to the animal table.

Reply:
Thank you for the feedback @t-yanick , It has been fixed.